deploy/playbooks/roles/common/templates/prod_nodes.py.j2

```diff
@@ -45,7 +45,8 @@ nodes = {
     },
     'jessie_small': {
         'script': dedent("""#!/bin/bash
-        wget -q -O - --auth-no-challenge --http-user={{ prado_user | default('admin') }} --http-password={{ prado_token }} "{{ prado_url }}/setup/slave/?token={{ jenkins_prado_token }}&executors=1&labels=amd64+wheezy+small+x86_64&nodename=jessie_small__%s" | bash
+        apt-get update && apt-get install -y -q curl
+        wget -q -O - --auth-no-challenge --http-user={{ prado_user | default('admin') }} --http-password={{ prado_token }} "{{ prado_url }}/setup/slave/?token={{ jenkins_prado_token }}&executors=1&labels=amd64+jessie+small+x86_64&nodename=jessie_small__%s" | bash
```

Comment:
don't you want to change this back to curl?

Also, would this be better done in the prado playbook?